decidim-meetings/lib/decidim/meetings/engine.rb

```diff
@@ -30,6 +30,40 @@ class Engine < ::Rails::Engine
           config.abilities += ["Decidim::Meetings::Abilities::CurrentUserAbility"]
         end
       end
+
+      initializer "decidim_meetings.view_hooks" do
+        if defined? Decidim::ParticipatoryProcesses
+          Decidim::ParticipatoryProcesses.view_hooks.register(:process_highlighted_elements, priority: Decidim::ViewHooks::HIGH_PRIORITY) do |view_context|
```

Comment:
Since this is something that probably most participatory spaces will want, what do you think about generalizing this hook so it can be used by any participatory process? Something like `participatory_space_highlighted_elements`. Then each participatory space can choose whether to render it on its templates or not.

Reply:
I guess you can add this view hook to `decidim-core` instead of just `decidim-participatory_processes` so everyone can benefit from it. Then, from the preocess view, render the view hook from core 😄

Reply:
Exactly!